src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -305,6 +307,24 @@ private void execute() {
                 buildId.ifPresent(value ->
                         buildScan.value("CI build number", value));
             }
+
+            if (isBuildkite()) {
+                envVariable("BUILDKITE_BUILD_URL").ifPresent(url ->
+                        buildScan.link("Buildkite build", url));
+                envVariable("BUILDKITE_COMMAND").ifPresent(command ->
+                        addCustomValueAndSearchLink(buildScan, "CI command", command));
+                envVariable("BUILDKITE_BUILD_ID").ifPresent(id ->
+                        buildScan.value("CI build ID", id));
+
+                Optional<String> buildkitePrRepo = envVariable("BUILDKITE_PULL_REQUEST_REPO");
+                Optional<String> buildkitePrNumber = envVariable("BUILDKITE_PULL_REQUEST");
+                if (Stream.of(buildkitePrRepo, buildkitePrNumber).allMatch(Optional::isPresent)) {
```

Comment:
is this line the same as in the Gradle plugin?

Reply:
Heh, yeah, I'll fix it 👍